### ISSUE TITLE
Foundation_vs170.vcxproj doesn't compile with out of the box VS 2022 components

### ISSUE THREAD

**Issue by hmartinez82** [pocoproject/poco#3605]
The Foundation project is asking for the Windows 8.1 SDK, but VS 2022 comes with Windows 10 SDK. This bug seems to be only present in the  Foundation project.

The build errors out with:

```
Build started...
1>------ Build started: Project: Foundation, Configuration: debug_static_md x64 ------
1>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VC\v170\Microsoft.Cpp.WindowsSDK.targets(46,5): error MSB8036: The Windows SDK version 8.1 was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution".
1>Done building project "Foundation_vs170.vcxproj" -- FAILED.
```

**Comment by aleks-f** [comment]
@hmartinez82 I'm aware that Foundation does not have progen file, I think there was a reason for it, but I do not remember what it was. @obiltschnig would know.

**Comment by hmartinez82** [comment]
@aleks-f I'm curious about the reason, but since there's no progen, then I'll fix the solution for this issue and for #3486 manually
